src/main/java/org/jenkinsci/plugins/workflow/support/steps/ExecutorStepExecution.java

```diff
@@ -389,11 +389,7 @@ private static void cancelOwnerExecution(Node node, CauseOfInterruption... cause
                             return;
                         }
                         listener.getLogger().println("Agent " + node.getNodeName() + " was deleted; cancelling node body");
-                        if (Util.isOverridden(BodyExecution.class, body.getClass(), "cancel", Throwable.class)) {
-                            body.cancel(new FlowInterruptedException(Result.ABORTED, false, causes));
-                        } else { // TODO remove once https://github.com/jenkinsci/workflow-cps-plugin/pull/570 is widely deployed
```

Comment:
https://github.com/jenkinsci/workflow-cps-plugin/pull/570 was released over two years ago now, so this is likely safe enough.